server/utils/roleHelpers.ts

```diff
@@ -3,10 +3,10 @@ import { userCanEdit, userHasAllRoles } from './utils'
 import { Prisoner } from '../interfaces/prisoner'
 import { User } from '../data/hmppsAuthClient'
 
-export const canViewCaseNotes = (user: User, prisoner: Prisoner) => {
+export const canViewCaseNotes = (user: User, prisoner: Partial<Prisoner>) => {
   return userHasAllRoles([Role.GlobalSearch, Role.PomUser], user.userRoles) || userCanEdit(user, prisoner)
 }
 
-export const canAddCaseNotes = (user: User, prisoner: Prisoner) => {
+export const canAddCaseNotes = (user: User, prisoner: Partial<Prisoner>) => {
   return userHasAllRoles([Role.GlobalSearch, Role.PomUser], user.userRoles) || userCanEdit(user, prisoner)
 }
```

Comment:
Sorry, think I badly advised here - should have probably been `Pick<Prisoner, 'prisonId' | 'restrictedPatient' ...whatever else it needs>` to ensure type safety